uniflow/flow/config.py

```diff
@@ -134,6 +134,70 @@ class TransformCopyConfig(TransformConfig):
     )
     model_config: ModelConfig = field(default_factory=lambda: {})
 
+class TransformForClassificationOpenAIGPT3p5Config(TransformConfig):
+    flow_name: str = "TransformOpenAIFlow"
+    model_config: ModelConfig = field(
+        default_factory=lambda: OpenAIModelConfig(
+            model_name="gpt-3.5-turbo-1106",
```

Comment:
qq: why 3.5 for this config and 4 for clustering config?

Reply:
bc gpt 4 is expensive? clustering needs both sematics classification and remove similar words, maybe gpt4 is better?